src/pages/MyCoursePage.tsx

```diff
@@ -0,0 +1,327 @@
+                        description: 'Test #1',
+                        weight: 30,
+                        type: 'Mündliche Prüfung',
+                    },
+                    {
+                        id: 2,
+                        description: 'Test #2',
+                        weight: 60,
+                        type: 'Schriftliche Prüfung',
+                    },
+                    {
+                        id: 3,
+                        description: 'Test #3',
+                        weight: 10,
+                        type: 'Mündliche Prüfung',
+                    },
+                ],
+                teachers: [
+                    {
+                        id: 2,
+                        name: 'John Doe',
+                    },
+                ],
+            },
+            {
+                id: 2,
+                description: 'Objektorientierte Programmierung II',
+                exams: [
+                    {
+                        id: 4,
+                        description: 'Test #1',
+                        weight: 40,
+                        type: 'Mündliche Prüfung',
+                    },
+                    {
+                        id: 5,
+                        description: 'Test #2',
+                        weight: 20,
+                        type: 'Mündliche Prüfung',
+                    },
+                    {
+                        id: 6,
+                        description: 'Test #3',
+                        weight: 40,
+                        type: 'Schriftliche Prüfung',
+                    },
+                ],
+                teachers: [
+                    {
+                        id: 2,
+                        name: 'John Doe',
+                    },
+                ],
+            },
+            {
+                id: 3,
+                description: 'Software-Architektur',
+                exams: [
+                    {
+                        id: 7,
+                        description: 'Test #1',
+                        weight: 50,
+                        type: 'Schriftliche Prüfung',
+                    },
+                    {
+                        id: 8,
+                        description: 'Test #2',
+                        weight: 30,
+                        type: 'Mündliche Prüfung',
+                    },
+                    {
+                        id: 9,
+                        description: 'Test #3',
+                        weight: 20,
+                        type: 'Mündliche Prüfung',
+                    },
+                ],
+                teachers: [
+                    {
+                        id: 2,
+                        name: 'John Doe',
+                    },
+                ],
+            },
+        ],
+    },
+];
+
+export default function MyCoursePage() {
+    const dropdownItems = customCourses.map((customCourse, index) => ({
+        label: customCourse.startDate,
+        value: index,
+    }));
+
```

Comment:
This variable will get redefined on every render. If you do not need any state or props (which is the case here) put this variable outside the component so it is only created once